be/src/column/column_pool.h

```diff
@@ -266,7 +266,6 @@ class CACHELINE_ALIGNED ColumnPool {
         if (lp) {
             _local_pool = nullptr;
             butil::thread_atexit_cancel(LocalPool::delete_local_pool, lp);
-            delete lp;
```

Comment:
Why delete this?

Reply:
as `butil::thread_atexit_cancel(LocalPool::delete_local_pool, lp);` will delete it later.

Reply:
`butil::thread_atexit_cancel(LocalPool::delete_local_pool, lp)` is used to remove the registered function `LocalPool::delete_local_pool`, so `LocalPool::delete_local_pool(lp)` will not be called.

Reply:
my mistake, ignore the `cancel`